lib/alerts_viewer_web/live/alerts_to_close_live.ex

```diff
@@ -0,0 +1,108 @@
+defmodule AlertsViewerWeb.AlertsToCloseLive do
+  @moduledoc """
+  LiveView for displaying alerts that we can maybe close.
+  """
+  use AlertsViewerWeb, :live_view
+  alias Alerts.Alert
+  alias Routes.{Route, RouteStats, RouteStatsPubSub}
+  alias TripUpdates.TripUpdatesPubSub
+
+  @impl true
+  def mount(_params, _session, socket) do
+    stop_recommendation_algorithm_components =
+      Application.get_env(:alerts_viewer, :stop_recommendation_algorithm_components)
+
+    algorithm_options = algorithm_options(stop_recommendation_algorithm_components)
+    current_algorithm = hd(stop_recommendation_algorithm_components)
+
+    bus_alerts =
+      if(connected?(socket),
+        do: Alerts.subscribe(),
+        else: []
+      )
+
+    stats_by_route = if(connected?(socket), do: RouteStatsPubSub.subscribe(), else: %{})
+
+    alerts_by_route = alerts_by_route(bus_alerts)
+
+    block_waivered_routes = if(connected?(socket), do: TripUpdatesPubSub.subscribe(), else: [])
+
+    socket =
+      assign(socket,
+        algorithm_options: algorithm_options,
+        current_algorithm: current_algorithm,
+        stats_by_route: stats_by_route,
+        block_waivered_routes: block_waivered_routes,
+        alerts_by_route: alerts_by_route,
+        routes_with_recommended_closures: []
+      )
+
+    {:ok, socket}
+  end
+
+  @impl true
+  def handle_info({:alerts, alerts}, socket) do
+    alerts_by_route = alerts_by_route(alerts)
+    {:noreply, assign(socket, alerts_by_route: alerts_by_route)}
+  end
+
+  @impl true
+  def handle_info({:stats_by_route, stats_by_route}, socket) do
+    {:noreply, assign(socket, stats_by_route: stats_by_route)}
+  end
+
+  @impl true
+  def handle_info({:block_waivered_routes, block_waivered_routes}, socket) do
+    {:noreply, assign(socket, block_waivered_routes: block_waivered_routes)}
+  end
+
+  @impl true
+  def handle_info(
+        {:updated_routes_with_recommended_closures, routes_with_recommended_closures},
+        socket
+      ) do
+    socket =
+      assign(socket,
+        routes_with_recommended_closures: routes_with_recommended_closures
+      )
+
+    {:noreply, socket}
+  end
+
+  @impl true
+  def handle_event("select-algorithm", %{"algorithm" => module_str}, socket) do
+    current_algorithm = String.to_atom(module_str)
+    {:noreply, assign(socket, current_algorithm: current_algorithm)}
+  end
+
+  defp alerts_by_route(alerts) do
+    alerts
+    |> filtered_by_bus()
+    |> filtered_by_delay_type()
+    |> Alerts.by_route()
+    |> Map.to_list()
+    |> Enum.map(fn {head, tail} -> {String.to_atom(head), tail} end)
```

Comment:
```suggestion
    |> Enum.map(fn {route_id, alerts} -> {String.to_atom(route_id), alerts} end)
```
nitpick: If I'm following the data correctly, I _think_ that these names might be more descriptive?